lib/integration/Target.js

```diff
@@ -158,26 +158,83 @@ export default class Target extends Plugin {
 
   async install ({ clone = false } = {}) {
```

Comment:
the install function is quite lengthy, can any of this be separated out into individual functions?